kong/runloop/plugin_servers/init.lua

```diff
@@ -329,16 +340,12 @@ end
 
 
 function plugin_servers.start()
-  if worker_id() ~= 0 then
-    kong.log.notice("only worker #0 can manage")
-    return
-  end
-
-  local pluginserver_timer = proc_mgmt.pluginserver_timer
+  local handler = worker_id() == 0 and proc_mgmt.pluginserver_timer
```

Comment:
let's create a new function, like `plugin_servers.check()`, that runs on all worker, and only call `plugin_servers.start()` as worker 0. this will make code more straight forward.